### PR TITLE
Stops MULEs from running over SSDs

### PR DESCRIPTION
**Pull request by TheSardele**
## What Does This PR Do
Stops SSD people from being run over by MULEbots.

## Why It's Good For The Game
Because they usually just get an admin rejuvenation anyway, may as well cut out the need for admin intervention.

## Changelog
:cl:
tweak: MULEbots no longer cause any damage when running over SSDs.
/:cl:

### CODE REVIEW

**Comment by FreeStylaLT** [comment]
no fun allowed 😞

**Comment by improvedname** [comment]
But what if i log out as soon a mulebot knocks me over?

**Comment by Ty-Omaha** [comment]
more of a tweak than a delete

File: code/modules/mob/living/simple_animal/bot/mulebot.dm [hunk 1]
```diff
@@ -696,34 +696,37 @@
 	return ..()
 
 /mob/living/simple_animal/bot/mulebot/proc/RunOver(mob/living/carbon/human/H)
-	add_attack_logs(src, H, "Run over (DAMTYPE: [uppertext(BRUTE)])")
-	H.visible_message("<span class='danger'>[src] drives over [H]!</span>", \
-					"<span class='userdanger'>[src] drives over you!</span>")
-	playsound(loc, 'sound/effects/splat.ogg', 50, 1)
+	if(!H.player_logged)//No running over SSD people
```
Comment (variableundefined) [inline code/modules/mob/living/simple_animal/bot/mulebot.dm]: Early return is the preferred method of doing this instead of tabbing the whole thing.

Comment (TheSardele) [reply code/modules/mob/living/simple_animal/bot/mulebot.dm]: Fixed.

**Pull request merged**